src/aatoolbox/utils/raster.py

```diff
@@ -0,0 +1,447 @@
+
+        If data array already has correct coordinate ordering, it is
+        directly returned. Function largely copied from
+        https://github.com/perrygeo/python-rasterstats/issues/218.
+
+        Parameters
+        ----------
+        inplace : bool, optional
+            If True, will overwrite existing data array. Default is False.
+
+        Returns
+        -------
+        Union[xarray.DataArray, xarray.Dataset]
+            Dataset or data array with correct coordinate ordering.
+
+        Examples
+        --------
+        >>> import xarray
+        >>> import numpy
+        >>> da = xarray.DataArray(
+        ...  numpy.arange(16).reshape(4,4),
+        ...  coords={"lat":numpy.array([87, 88, 89, 90]),
+        ...          "lon":numpy.array([70, 69, 68, 67])}
+        ... )
+        >>> da_inv = da.aat.invert_coordinates()
+        >>> da_inv.get_index("lon")
+        Int64Index([67, 68, 69, 70], dtype='int64', name='lon')
+        >>> da_inv.get_index("lat")
+        Int64Index([90, 89, 88, 87], dtype='int64', name='lat')
+        """
+        data_obj = self._get_obj(inplace=inplace)
+        lon_inv, lat_inv = self._check_coords_inverted()
+
+        # Flip the raster as necessary (based on the flags)
+        inv_dict = {}
+
+        if lon_inv:
+            logger.info("Longitude was inverted, reversing coordinates.")
+            inv_dict[self.x_dim] = data_obj[self.x_dim][::-1]
+
+        if lat_inv:
+            logger.info("Latitude was inverted, reversing coordinates.")
+            inv_dict[self.y_dim] = data_obj[self.y_dim][::-1]
+
+        if inv_dict:
+            data_obj = data_obj.reindex(inv_dict)
+
+        return data_obj if not inplace else None
+
+    def _check_coords_inverted(self):
+        """
+        Check if latitude and longitude inverted.
+
+        Examples
+        --------
+        >>> import xarray
+        >>> import numpy
+        >>> da = xarray.DataArray(
+        ...  numpy.arange(16).reshape(4,4),
+        ...  coords={"lat":numpy.array([90, 89, 88, 87]),
+        ...          "lon":numpy.array([70, 69, 68, 67])}
+        ... )
+        >>> da.aat._check_coords_inverted()
+        (True, False)
+        """
+        data_obj = self._get_obj(inplace=False)
+        lat = data_obj.get_index(self.y_dim)
+        lat_start = lat[0]
+        lat_end = lat[-1]
+        lon = data_obj.get_index(self.x_dim)
+        lon_start = lon[0]
+        lon_end = lon[-1]
+        return lon_start > lon_end, lat_start < lat_end
+
+    def change_longitude_range(
+        self, inplace: bool = False
+    ) -> xarray.DataArray:
+        """Convert longitude range between -180 to 180 and 0 to 360.
+
+        For some raster data, outputs are incorrect if longitude ranges
+        are not converted from 0 to 360 to -180 to 180, such as the IRI
+        seasonal forecast, but *not* the IRI CAMS observational terciles,
+        which are incorrect if ranges are stored as -180 to 180.
```

Comment:
Honestly this CAMS thing, I would have to double-check as I was doing that in the first weeks I learned about geospatial processing. Haven't used the functionality structurally for anything else than IRI. And it is more standard to use -180 to 180 so I am doubting if we should even provide functionality to convert to 0-360 or that might cause incorrect use. 

I will add to my to do to look into it a bit

Reply:
Okay thanks @Tinkaa, will add as a separate issue to ensure proper tracking. Will leave in for now!